### PR TITLE
MAUI-1692-UGDocumentMAUI-Documentation updated.

### PR DESCRIPTION
**Pull request by Indumathi1195R**
Documentation updated.

### CODE REVIEW

File: MAUI/Scheduler/migration.md [hunk 1]
```diff
@@ -22,6 +22,70 @@ To make the migration from the Xamarin [Xamarin SfSchedule](https://www.syncfusi
...
+<th>Xamarin SfSchedule</th>
+<th>.NET MAUI SfScheduler</th></tr>
+<tr>
+<td>
+{% tabs %}
+
+{% highlight xaml %}
+
+<ContentPage
+xmlns:scheduler="clr-namespace:Syncfusion.Maui.Scheduler;assembly=Syncfusion.Maui.Scheduler">
```
Comment (Jeyasri-Murugan) [inline MAUI/Scheduler/migration.md:40]: Add proper namepace for xamarin , seems used maui code

File: MAUI/Scheduler/migration.md [hunk 2]
```diff
@@ -1280,4 +1403,11 @@ To make the migration from the Xamarin [Xamarin SfSchedule](https://www.syncfusi
 * Command support for Tapped, DoubleTapped, LongPressed and ViewChanged events.
 * Support to change text color for each scheduler appointment.
 * Suspend and resume for appointment update.
-* Appointment Drag and Drop.
\ No newline at end of file
+* Appointment Drag and Drop.
```
Comment (Jeyasri-Murugan) [inline MAUI/Scheduler/migration.md:1406]: Can we add Appointment and settings code examples

File: MAUI/Scheduler/migration.md (hunk 1 above)
Comment (MuniappanSubramanian) [reply MAUI/Scheduler/migration.md]: Xamarin namespace added correctly,

File: MAUI/Scheduler/migration.md [hunk 3]
```diff
@@ -199,6 +263,66 @@ To make the migration from the Xamarin [Xamarin SfSchedule](https://www.syncfusi
 ## Properties
 
 ### SfSchedule
+
+The following code example, explains how to initialize the schedule in Xamarin SfSchedule and .NET MAUI SfScheduler.
```
Comment (Jeyasri-Murugan) [inline MAUI/Scheduler/migration.md:267]: The following code example, explains how to initialize the properties of the schedule in

**Review by Jeyasri-Murugan — APPROVED** [review]

**Comment by MuniappanSubramanian** [comment]
rebuild

**Pull request merged**